tests/integration/test_storage_kafka/test.py

```diff
@@ -229,18 +228,25 @@ def test_kafka_consumer_hang(kafka_cluster):
     time.sleep(0.5)
     kafka_cluster.unpause_container('kafka1')
 
+    # print("Attempt to drop")
     instance.query('DROP TABLE test.kafka')
 
+    #kafka_cluster.open_bash_shell('instance')
+
     instance.query('''
         DROP TABLE test.consumer;
         DROP TABLE test.view;
     ''')
 
-    log = '/var/log/clickhouse-server/stderr.log'
-    instance.exec_in_container(['grep', '-q', 'BROKERFAIL', log])
-    instance.exec_in_container(['grep', '-q', '|ASSIGN|', log])
-    instance.exec_in_container(['grep', '-q', 'Heartbeat failed: REBALANCE_IN_PROGRESS: group is rebalancing', log])
-    instance.exec_in_container(['grep', '-q', 'Group "consumer_hang": waiting for rebalance_cb', log])
+    # we expect no hanging drop queries
+    # 'dr'||'op' to avoid self matching
+    assert int(instance.query("select count() from system.processes where position(lower(query),'dr'||'op')>0")) == 0
```

Comment:
so this is a replacement for the `grep` below? maybe both should be leaved?

Reply:
discussed in telegram. 
that message doesn't appear in logs after the fix.

will add other check to ensure the test is doing what expected.